pkg/service/repair/progress_integration_test.go

```diff
@@ -25,6 +26,10 @@ import (
 	"github.com/scylladb/scylla-manager/v3/pkg/util/uuid"
 )
 
+func fhkt(host, keyspace, table string) hkt {
```

Comment:
What `fhkt` means ? 
How should I interpret this name seeing it at the first time ?

Reply:
It's just a 1-line wrapper to reduce test code length, but it can be removed/renamed in favor of more readability.